cmd/exporters/influxdb/influxdb_test.go

```diff
@@ -41,18 +46,55 @@ func TestAddrParameter(t *testing.T) {
 // test that the addr (and port) parameters
 // are handled properly to construct server URL
 func TestUrlParameter(t *testing.T) {
+	expectedURL := "https://some-valid-domain-name.net:8888/api/v2/write?org=netapp&bucket=harvest&precision=s"
+
+	opts := &options.Options{}
+	opts.Debug = true
+	var exporters map[string]conf.Exporter
+	exporterName := "influx-test-url"
+	var err error
+
+	path := "../../tools/doctor/testdata/testConfig.yml"
```

Comment:
I should have mentioned this yesterday, but this block of seven or so lines, that is repeated in all the tests, could be refactored to a single method they all call.

Reply:
Yes. Done.